### PR TITLE
Api updates

### PR DESCRIPTION
**Pull request by niklabh**
- Updated types with latest metadata
- Updated api version

### CODE REVIEW

File: examples/example.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
+const {
```
Comment (bobo-k2) [inline examples/example.ts]: Can you put some general comment on top, and describe what is this example about

File: packages/types/scripts/updateMetadata.ts [hunk 2]
```diff
@@ -1,19 +1,17 @@
...
+  const endpoint = 'wss://rpc.shibuya.astar.network';
   const ws = new WebSocket(endpoint);
   ws.onopen = (): void => {
     ws.send('{"id":"1","jsonrpc":"2.0","method":"state_getMetadata","params":[]}');
   };
   ws.onmessage = (msg: any): void => {
-    const metadata = JSON.parse(msg.data).result;
-    fs.writeFileSync('packages/types/src/metadata/static-latest.ts', `export default '${metadata}'`);
-    console.log('Done');
+    // const metadata = JSON.parse(msg.data).result;
```
Comment (bobo-k2) [inline packages/types/scripts/updateMetadata.ts]: Delete if not needed

**Review by bobo-k2 — CHANGES_REQUESTED** [review]

**Review by impelcrypto — APPROVED** [review]

**Comment by impelcrypto** [comment]
@niklabh Linting check is failed

File: examples/example.ts (hunk 1 above)
Comment (niklabh) [reply examples/example.ts]: sorry this file was added by mistake. removed

**Pull request merged**